main.js

```diff
@@ -74,7 +74,7 @@ function firstInstallTour(win) {
   if (win.activeInstall) {
     win.activeInstall = false;
     let document = win.document;
-    let button = document.getElementById('side-tabs-button');
+    let sidetabsbutton = document.getElementById('side-tabs-button');
```

Comment:
Since I'm using it more often - I didn't notice it when it didn't have a descriptive name. which is why on line 119 it was previously fetched from the dom again rather than reused.